core/backend/src/ElementAspect.ts

```diff
@@ -186,8 +186,7 @@ export class ExternalSourceAspect extends ElementMultiAspect {
     this.jsonProperties = props.jsonProperties;
   }
 
-  /**  Look up the ElementId of the element that contains an aspect with the specified Scope, Kind, and Identifier
-   */
+  /** @deprecated findAllBySource */
```

Comment:
Please add to the [deprecations section of NextVersion.md](https://github.com/iTwin/itwinjs-core/blob/master/docs/changehistory/NextVersion.md#deprecations).